### PR TITLE
Add minimum LogLevel to AssemblyTestLog

### PR DESCRIPTION
**Pull request by BrennanConroy**
MusicStore and SignalR would need to be updated

### CODE REVIEW

**Comment by muratg** [comment]
*ping*

**Comment by anurse** [comment]
*pong*

**Comment by jkotalik** [comment]
@BrennanConroy

File: src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs [hunk 1]
```diff
@@ -33,9 +33,9 @@ private AssemblyTestLog(ILoggerFactory globalLoggerFactory, ILogger globalLogger
             _assemblyName = assemblyName;
         }
 
-        public IDisposable StartTestLog(ITestOutputHelper output, string className, out ILoggerFactory loggerFactory, [CallerMemberName] string testName = null)
+        public IDisposable StartTestLog(ITestOutputHelper output, string className, out ILoggerFactory loggerFactory, LogLevel minLogLevel = LogLevel.Debug, [CallerMemberName] string testName = null)
```
Comment (JunTaoLuo) [inline src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs]: Don't we usually add overloads instead of using default parameters?

Comment (BrennanConroy) [inline src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs]:
```
public IDisposable StartTestLog(ITestOutputHelper output, string className, out ILoggerFactory loggerFactory, LogLevel minLogLevel = LogLevel.Debug, [CallerMemberName] string testName = null);

public IDisposable StartTestLog(ITestOutputHelper output, string className, out ILoggerFactory loggerFactory, [CallerMemberName] string testName = null);

// which one does it call? Ambiguous
TestOutputHelper.StartTestLog("className", out var factory);
```

Comment (JunTaoLuo) [inline src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs]: Not ambiguous.

**Pull request merged**